src/services/expenses.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+let expenses = [];
+
+const getAllExpenses = () => {
+  return expenses;
+};
+
+const getExpenseById = (expenseId) => {
+  const foundUser = expenses.find(expense => expense.id === expenseId);
+
+  return foundUser || null;
+};
+
+const addNewExpense = (
+  userId,
+  spentAt,
+  title,
+  amount,
+  category,
+  note,
+) => {
```

Comment:
You can just accept an object and then use the spread operator when creating a new expense.